### PR TITLE
Updating markdown files

### PR DESCRIPTION
**Pull request by cwebster-99**
- Updating README.md 
- Updating SUPPORT.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-
-As the maintainer of this project, please make a few updates:
-
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+# Python environment tools for Visual Studio Code
+
+Performant Python environment tooling and support, such as locating all global Python installs and virtual environments. Powers environment environment tooling for the [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) in [Visual Studio Code](https://code.visualstudio.com/).
```
Comment (cwebster-99) [inline README.md:3]: Perhaps we should exclude the second sentence since for now since that is not the current reality.

**Review by DonJayamanne — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (karthiknadig) [reply README.md:3]: I agree

File: README.md [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+## Environment Types Supported 
+- Windows Registry
+- Windows Store
+- PyEnv
+- Conda
+- PipEnv
+- Homebrew
+- VirtualEnvWrapper
+- Venv
+- VirtualEnv
```
Comment (karthiknadig) [inline README.md:14]:
```suggestion
- python.org
- Windows Store
- PyEnv
- PyEnv-Win
- PyEnv-Virtualenv
- Conda
- PipEnv
- Homebrew
- VirtualEnvWrapper
- VirtualEnvWrapper-Win
- Venv
- VirtualEnv
- Python on your PATH
```

**Review by karthiknadig — CHANGES_REQUESTED** [review]

**Comment by karthiknadig** [comment]
@DonJayamanne it also handles `mamba` `micromamba` `miniconda` `miniforge` etc right?

**Comment by cwebster-99** [comment]
> @DonJayamanne it also handles `mamba` `micromamba` `miniconda` `miniforge` etc right?

That looks to be correct - I updated your edit above to include those. Is there a place that references all of the supported environments I could pull from?

**Comment by DonJayamanne** [comment]
> I updated your edit above to include those. Is there a place that references all of the supported environments I could pull from?

I'd say yes, but anaconda, micromamba, conda, etc. are all different installs and thats not easily discoverable for documentation purposes.

> t also handles mamba micromamba miniconda miniforge etc right?

Not really, I haven't added specific code to look for micromamba, 
Will add a PR to look for some of the known paths.

**Review by DonJayamanne — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+
+## Features 
+
+- Discovery of all global Python installs
+- Discovery of all Python virtual environments
+
+## Key Methodology
+
+We leverage Rust's performance to quickly scan for Python environments available to the user. When we find them, we gather all relevant information immediately to avoid the need for repeated discovery.
```
Comment (cwebster-99) [inline README.md:24]: Would like to call out out methodology and "principles" used in this work. Will likely need some additional details to ensure this is comprehensive

File: README.md [hunk 4]
```diff
@@ -1,14 +1,35 @@
...
+- Miniconda
+- Miniforge
+- PipEnv
+- Homebrew
+- VirtualEnvWrapper
+- VirtualEnvWrapper-Win
+- Venv
+- VirtualEnv
+- Python on your PATH
+- Mamba
```
Comment (DonJayamanne) [inline README.md:22]: Mamba now recomments using `miniforge or micromamba`, hence I'd not mention this.

File: README.md [hunk 5]
```diff
@@ -1,14 +1,35 @@
...
+- Micromamba
```
Comment (DonJayamanne) [inline README.md:23]: Lets leave this out for now, its not easy to get the exe for micromamba,
We can add this later.
We might end up discovering the envrionments, but perhaps not the conda exe.

File: README.md (hunk 4 above)
Comment (DonJayamanne) [reply README.md:22]: Please remove this section

File: README.md (hunk 5 above)
Comment (DonJayamanne) [reply README.md:23]: Please remove this line

Comment (karthiknadig) [inline README.md:23]:
```suggestion
```

**Review by DonJayamanne — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -1,14 +1,33 @@
...
+- Python on your PATH
+
+## Features 
+
+- Discovery of all global Python installs
+- Discovery of all Python virtual environments
+
+## Key Methodology
+
+We leverage Rust's performance to quickly scan for Python environments available to the user. When we find them, we gather all relevant information immediately to avoid the need for repeated discovery.
```
Comment (cwebster-99) [inline README.md:30]:
```suggestion
Our approach is focused on performance and efficiency which leverages Rust to achieve those goals. We limit the number of I/O operations done in order to obtain relevant environment information and reduce the number of repeated I/O operations needed by gathering all relevant environment information immediately. This also reduces the need to spawn processes helping increase performance. 
```

**Review by karthiknadig — APPROVED** [review]

**Pull request merged**